src/common/utils.js

```diff
@@ -263,15 +263,14 @@ const getCardColors = ({
  */
 const wrapTextMultiline = (text, width = 59, maxLines = 3) => {
```

Comment:
I am not sure about the function name with this change; it's now returning not only a wrapped version of a multiline text but an encoded and wrapped version. Maybe this doesn't matter in this case, but I just want to point it out in case it does